2022/39xxx/CVE-2022-39224.json

```diff
@@ -1,18 +1,93 @@
 {
```

Comment:
' https://github.com/jordansissel/ruby-arr-pm/pull/14' does not match '^(ftp|http)s?://\\S+$'